pathd/path_pcep_pcc.c

```diff
@@ -542,9 +547,11 @@ void pcep_pcc_pcep_event_handler(struct ctrl_state *ctrl_state,
 		PCEP_DEBUG_PCEP("%s PCEP message: %s", pcc_state->tag,
 				format_pcep_message(event->message));
 		break;
+	case PCE_DEAD_TIMER_EXPIRED:
+		PCEP_DEBUG("%s PCE_DEAD_TIMER_EXPIRED %d", pcc_state->tag,
```

Comment:
The received even is already logged before, this is only to have the extra retry counter. I am not sure we need that extra logging. I assume you added it for debug and it wasn't cleaned up.

Reply:
yes, deleted